echopype/tests/test_echodata.py

```diff
@@ -59,7 +59,7 @@ def test_constructor(self):
             'vendor',
         ]
 
-        assert ed.sonar_model == 'EK60'
+        # assert ed.sonar_model == 'EK60'
```

Comment:
This failed the tests. I don't think sonar_model should be "EK60"; it's not being passed to the constructor.

Reply:
FYI, this must be due to this [bug in EK60 sonar_model parsing](https://github.com/OSOceanAcoustics/echopype/issues/210#issuecomment-821761942), where "ER60" (the software that runs EK60) is being entered by mistake. Ultimately we need to address that issue, but in the meantime I agree that it's better to comment out that assertion.Thanks! cc @leewujung

Reply:
Did the file `Summer2017-D20170615-T190214.zarr` sonar model changed? This test should have parsed the sonar model from there, which it used to be "EK60". @imranmaj added "keywords" in ds on https://github.com/OSOceanAcoustics/echopype/blob/5c8d1db8d48f6928532ff44a66b0991271301677/echopype/echodata/echodata.py#L137

Does this check potentially break the intended behavior?

Reply:
>Did the file Summer2017-D20170615-T190214.zarr sonar model changed? This test should have parsed the sonar model from there, which it used to be "EK60"

What I'm referring to seems to be a broader problem and is not specific to that file. See the brief discussion I pointed to. Though, are you saying that `assert ed.sonar_model == 'EK60'` used to pass with that file as a test?

Reply:
> Though, are you saying that assert ed.sonar_model == 'EK60' used to pass with that file as a test?

Yepp. See current build:
https://github.com/OSOceanAcoustics/echopype/runs/2822876705#step:12:241

Reply:
Looks like this is fixed. Thanks @imranmaj